app/views/NewMessageView.tsx

```diff
@@ -144,9 +146,15 @@ class NewMessageView extends React.Component<INewMessageViewProps, INewMessageVi
 	}
 
 	onSearchChangeText(text: string) {
+		this.setState({ searchText: text });
 		this.search(text);
 	}
 
+	handleCancel = () => {
```

Comment:
`handleCancel` what?
Try switching to `handleCancelSearch`, or `cancelSearch`, something like that.

Reply:
Yes definitely!